### ISSUE TITLE
Improve

### ISSUE THREAD

**Issue by omnituensaeternum** [AstraaDev/Discord-Token-Grabber-V2#1]
https://github.com/NobodyWouldEverUseThis7/Token-encryption-writeup

**Comment by omnituensaeternum** [comment]
Spastic fuck can't even skid properly lol.

**Comment by AstraaDev** [comment]
Who exactly are you talking about?

**Comment by AstraaDev** [comment]
HAHAHAHAH I don't know if the funniest thing is that you skid my AIO by making it pay (at crazy prices) or the fact that you come all the way here to tell your shit. Knowing that discord's encryption method has been known for months by everyone I didn't need to use this person's work to fix my script.
But keep on talking shit no worries, I'm not doing this for money (don't feel targeted) nor for recognition but to progress on my side.

On these nice words, good day to you haha!

**Comment by AstraaDev** [comment]
WordPress blogs xDDDDD

And even, they can think what they want, let them use my scripts as they want, I don't do that for others and if they talk about me on blogs (xDD) so much the better, it makes me more publicity.
What I find really crazy with you (without any bad pun), is that you manage to convince yourself so much, that you didn't skid my work. Nobody knows the existence of your tool or even your work (if you have ever worked once in your life) or even your name (me first) and you dare to come to my discord, and my GitHub projects to say such things.

**Comment by AstraaDev** [comment]
When I talk about your work I'm just talking about the Tool you stole from me. The rest I don't care about xD. This does not concern me and unlike some people I will not criticize the work of others just to enhance my own.

But whatever, the fact that you're taking it out on me hates me and makes me laugh a lot.

**Comment by AstraaDev** [comment]
Yes, it's quite a good thing to make people smile and pass on some good humor!

You say your work is from 2020 but no one on this planet has ever had proof of this. While I (who is not lying) started to create this tool exactly 1 year ago (with proof) but I published it only 1 month later on GitHub so that people can also use it easily.

**Comment by DestinyofYeet** [comment]
xD

**Comment by somecoulombs** [comment]
Lmao why did you steal this, that's fuckin lame

**Comment by GalaxzyDev** [comment]
Astra back at it again, least you could do is get your facts straight and give credit to Clara. But hey, guess that further proves her claim.
